Tasks/PowerShellV2/task.json

```diff
@@ -132,6 +132,15 @@
             "defaultValue": "",
             "helpMarkDown": "Working directory where the script is run.",
             "groupName": "advanced"
+        },
+        {
+            "name": "runScriptInSeparateScope",
+            "type": "boolean",
+            "label": "Allows to use Call operator - & for script execution",
```

Comment:
Nit: Could it make sense to change label also - to be more descriptive about goal of this input?

Reply:
Yeah, thank you I've changed the label for this input